### DISCUSSION TITLE
Project's end goal

### DISCUSSION THREAD

**Discussion by Shubhcoder** [alexrobaina/frontend_petsLove#199] [General]
Hi, @alexrobaina 

Can we have a general overview and workflow of the project? Maybe, a working prototype or figma design (whichever is suitable enough to clear the vision). As, it is getting hard to visualize the end result, making it hard to come up with further things (that are left to be done).

**Comment by alexrobaina** [comment]
Hi @Shubhcoder. 
It exists 😄 can you check the following link:
[Figma proptype](https://www.figma.com/file/MK1mSVutBHbVXEo4VyfzVf/Pets-love-system-design?type=design&node-id=3128%3A20335&mode=design&t=LLqQIY8zpVij5PRr-1)

Please tell me what you think.

**Reply by Shubhcoder** [reply]
Hey, thank you for the design link (this definitely helps to visualize the end result).

What do you say of having a tracking system, project's progress is tracked? This way I believe we can get this project ready faster, as then everyone will be on the same page.

Looking forward to your views on this.

**Comment by alexrobaina** [comment]
Hi @Shubhcoder! Thank you. 😄 
Well, about the Figma design, you probably noticed several inconsistencies because it's only a guide. 
For example: 
#### Figma design
![Screen Shot 2023-10-12 at 08 58 31](https://github.com/alexrobaina/frontend_petsLove/assets/30770802/ed275c85-6db3-4028-bfee-ae58d36bcaca)

#### Code implementation:
![Screen Shot 2023-10-12 at 08 57 27](https://github.com/alexrobaina/frontend_petsLove/assets/30770802/95ab127c-610e-493d-b158-79b83955ed35)

**Comment by alexrobaina** [comment]
And about  "Project's progress is tracked" please can you explain more about that. I mean do you have any examples or what kind of tools we can use to make the tracker?

Maybe you refer to: 
https://github.com/users/alexrobaina/projects/1/views/1

**Reply by Shubhcoder** [reply]
Hi, @alexrobaina 

Yes, I was referring to this project link that you shared.

**Comment by Shubhcoder** [comment]
Hi, @alexrobaina 

What's next from here in the frontend? Since, the figma designs are not exact reference (I am bit confused in what to do next). Also, the project view is having only one frontend task pending.

**Comment by alexrobaina** [comment]
Hi @Shubhcoder. 
I've started to create issues with all that we need to make to finish the project.

For now, I'm creating this issue to create the profile user: [issue](https://github.com/alexrobaina/frontend_petsLove/issues/209)
